### ISSUE TITLE
Group challenge wins and losses showing up in solo ranked scoreboard

### ISSUE THREAD

**Issue by Garamore1** [WarEmu/WarBugs#20504]
Group challenge wins and losses are showing up in solo ranked scoreboards.

To reproduce - get a 6 man and group challenge another - score will change in the ranked solo boards.

**Comment by lilimx** [comment]
Hey,

uhm is there already an example?

**Comment by Garamore1** [comment]
Yes - did some group challenges last night and if you check garamar on the solo ranked scoreboard there are results - havent done a single solo ranked as they arent popping.
